### DISCUSSION TITLE
simplify menus shortcuts editing

### DISCUSSION THREAD

**Discussion by pmp-p** [phcode-dev/phoenix#970] [Ideas]
by adding the possibility to edit ( maybe with right mouse click ) actions directly from drop down menus or at least show a message box with the override name of targeted action and a widget that can display a user entered shortcut to help filling keymap.json.

i found that https://github.com/redmunds/brackets-display-shortcuts that may be related

**Comment by abose** [comment]
Thanks for the suggestion. Related to https://github.com/phcode-dev/phoenix/issues/147 please track progress in that issue.
